CRM/Core/BAO/MessageTemplate.php

```diff
@@ -376,7 +376,10 @@ protected static function renderTemplateRaw($params) {
    * @throws \CRM_Core_Exception
    * @throws \API_Exception
    */
-  public static function sendTemplate($params) {
+  public static function sendTemplate(array $params): array {
+    // Handle isEmailPdf here as the unit test on that function deems it 'non-conforming'.
+    $isAttachPDF = !empty($params['isEmailPdf']);
+    unset($params['isEmailPdf']);
```

Comment:
Just noting that by unsetting it it no longer will be in the params passed to the mail hooks, but that should be ok.

Reply:
Yeah - the test enforces that